scripts/check-storage.js

```diff
@@ -12,7 +12,8 @@ fs.readdirSync("./contracts/").forEach(contractName => {
       "EtherRouter.sol",
       "Migrations.sol",
       "Resolver.sol",
-      "Token.sol"
+      "Token.sol",
+      "ReputationMiningCycle.sol"
```

Comment:
No need to include it here as this is not an `EternalStorage`-based contract

Reply:
The build broke without it:
https://circleci.com/gh/JoinColony/colonyNetwork/1119

From what I understand:
We want to avoid state defined in contract that inherit from ColonyNetworkStorage. That's a way to centralize all the state in the ColonyNetworkStorage definition,
right?

If that's the case we might need to check the inheritance hierarchy in the `check-storage.js` script.

Reply:
Sorry the diff was hiding the part that this is an exclusion list. You've correctly excluded it, my bad.